fix(docs): tutorials tags cleanup j to o (#3307)
scaleway/docs-content: tutorials/jitsi-docker/index.mdx

  paragraph: Docker allows you to deploy a self-hosted Jitsi Meet videoconferencing solution in a few easy steps.
categories:
  - compute
tags: media Jitsi-Meet videoconferencing
dates:
  validation: 2023-12-13
  posted: 2020-03-24
---

> [Jitsi Meet](https://jitsi.org/jitsi-meet/) is a fully encrypted, open-source video conferencing tool that does not require an account or subscription. The solution can be installed natively on [Ubuntu Bionic Beaver (18.04 LTS)](/tutorials/jitsi-ubuntu-bionic) and [Debian Buster](/tutorials/jitsi-debian). Another way to install it is using a containerized application running on [Docker](https://www.docker.com/).

Reviewer: ```suggestion
[Jitsi Meet](https://jitsi.org/jitsi-meet/) is a fully encrypted, open-source video conferencing. You do not need to have any account or subscription to use it. The solution can be installed natively on [Ubuntu Bionic Beaver (18.04 LTS)](/tutorials/jitsi-ubuntu-bionic) and [Debian Buster](/tutorials/jitsi-debian). You can also install it using a containerized application running on [Docker](https://www.docker.com/).
```

> [Jitsi Meet](https://jitsi.org/jitsi-meet/) is a fully encrypted, open-source video conferencing tool that does not require an account or subscription. The solution can be installed natively on [Ubuntu Bionic Beaver (18.04 LTS)](/tutorials/jitsi-ubuntu-bionic) and [Debian Buster](/tutorials/jitsi-debian). Another way to install it is using a containerized application running on [Docker](https://www.docker.com/).

Reviewer 2: ```suggestion
Jitsi Meet is a fully encrypted, open-source video conferencing tool that does not require an account or subscription. The solution can be installed natively on [Ubuntu Bionic Beaver (18.04 LTS)](/tutorials/jitsi-ubuntu-bionic) and [Debian Buster](/tutorials/jitsi-debian). Another way to install it is using a containerized application running on [Docker](https://www.docker.com/).
```
